### ISSUE TITLE
Broken for JDK 52 (Java 8) since 0.5.0

### ISSUE THREAD

**Issue by bsless** [gerritjvv/pjson#35]
Hello,
The title is pretty much self explanatory. Since the library was compiled vs Java 9 (JDK 53) I'm getting the following exception when loading the namespace:
```
pjson/ToJSONString has been compiled by a more recent version of the Java Runtime (class file version 53.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```
By rolling back to "0.4.0" I had no errors..
Is there any workaround or is the only solution building vs. Java 8?

Thanks

**Comment by gerritjvv** [comment]
hi. I think I did this as part of my work on making the lib compatible with java 1.9.
I'll try compile and release with 1.8.

**Comment by gerritjvv** [comment]
this is fixed now in 0.5.2.

**Comment by bsless** [comment]
Wonderful, thank you!
